### PR TITLE
Temporal solution for #101

### LINKED ISSUE #101: Integration test is flaky for status updates
see https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/90/pull-kueue-test-integration-main/1501214768197799936

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

#### What type of PR is this?

/kind bug

#### Which issue(s) this PR fixes:

Fixes #101

#### Special notes for your reviewer:
🤞🏽

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
wiiiiii

**Comment by ArangoGutierrez** [comment]
4
/test pull-kueue-test-integration-main

**Comment by ArangoGutierrez** [comment]
this patch takes #101 down to 20% (1/5)

**Comment by ArangoGutierrez** [comment]
6
/test pull-kueue-test-integration-main

**Comment by ArangoGutierrez** [comment]
7
/test pull-kueue-test-integration-main

File: pkg/controller/core/queue_controller.go [hunk 1]
```diff
@@ -131,23 +132,23 @@ type queuedWorkloadHandler struct{}
 func (h *queuedWorkloadHandler) Create(e event.CreateEvent, q workqueue.RateLimitingInterface) {
 	qw := e.Object.(*kueue.QueuedWorkload)
 	if qw.Spec.Admission == nil {
-		q.Add(requestForQueueStatus(qw))
+		q.AddAfter(requestForQueueStatus(qw), time.Second*1)
```
Comment (alculquicondor) [inline pkg/controller/core/queue_controller.go:135]: just `time.Second`

Also, can you do the same for the clusterqueue_controller?

**Comment by ahg-g** [comment]
please define a constant in the constants pkg and increase it to 3

File: pkg/constants/constants.go [hunk 2]
```diff
@@ -24,4 +26,10 @@ const (
 
 	ManagerName       = "kueue-manager"
 	JobControllerName = "kueue-job-controller"
+
+	// RequeueAfterDuration is the indicated time to wait before adding an item to the reconcile
+	// workqueue.
+	//
+	// TODO(#101)
```
Comment (alculquicondor) [inline pkg/constants/constants.go:33]: TODO what?

I suppose: "remove this"

Although it's useful to keep it long term as a batching mechanism.

So I would call the variable: "UpdatesBatchPeriod", similar to how we do it in the job controller:

https://github.com/kubernetes/kubernetes/blob/2b1b849d6a8bdeb7dc0807438cfd0ff2a9d752c1/pkg/controller/job/job_controller.go#L62

File: pkg/controller/core/queue_controller.go (hunk 1 above)
Comment (ahg-g) [reply pkg/controller/core/queue_controller.go]: I would something higher than a second, something higher would be more resilient and doesn't really impact the user experience.

File: pkg/constants/constants.go (hunk 2 above)
Comment (alculquicondor) [reply pkg/constants/constants.go]: I now, what you should say what you expect "to do" in the comment.

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**